docs/Antora/modules/howto_guides/pages/howto_add_function_block.adoc

```diff
@@ -161,20 +160,21 @@ device = instance.add_device('daqref://device0')
 
 # get available function block types
 function_block_types = instance.available_function_block_types
-for function_block_type in function_block_types.keys():
+for function_block_type in function_block_types:
     print(function_block_type)
 
 # if there is no statistics function block available, exit with an error
-if not "ref_fb_module_statistics" in function_block_types.keys():
-    self.assertTrue(False, "Function block not found")
+if 'ref_fb_module_statistics' not in function_block_types:
```

Comment:
I find it a bit odd that this, which is supposed to be the "full listing", was not a part of the document above... Perhaps this is still WIP?

Reply:
Some of the how-to articles are a minimal representation of a feature. They might include additional code required to make the listing a self-contained executable (runners are such an example).

Here, however, I do agree that the code here should either be removed, or also added above as an example.

Reply:
I don't think it's really WIP and this code has been there since July.

The text above explains the concepts, how to use the functions, and then says:
> If there is no such Function Block available, the method will fail.

So it does mention it in a way, just not with the exact code. This is a complete listing with some additional error-checking that was left out earlier (and is also not strictly needed). I don't have my favourite: both mentioning and not mentioning it explicitly seems equally fine, but I leave it up to @dejancrn / @JakaMohorkoDS to comment.

(But the error message could potentially also mention the name of the missing block.)